pkg/scalers/externalscaler/externalscaler.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.30.0
-// 	protoc        v4.23.2
+// 	protoc        v3.18.1
```

Comment:
Could you revert this change? (and also in other protoc files)

Reply:
Done